tests/test_dfa.py

```diff
@@ -1280,6 +1280,35 @@ def test_minify_partial_dfa(self) -> None:
         self.assertEqual(len(minified_partial_dfa.states), 4)
         self.assertEqual(minified_partial_dfa, partial_dfa_extra_state)
 
+    def test_minify_partial_dfa_correctness(self) -> None:
+        """Test correctness of minifying partial DFAs"""
```

Comment:
For further context, I would add a link to the GitHub issue

Reply:
Added comment here: https://github.com/caleb531/automata/pull/184/commits/ecd87c4e0350cf01e2d721ba5c61b7e9cd2a39f1